dask/tests/test_tokenize.py

```diff
@@ -264,13 +270,28 @@ def test_normalize_numpy_ufunc_unserializable():
     # Make a ufunc that isn't in the numpy namespace and can't be serialized
     inc = np.frompyfunc(lambda x: x + 1, 1, 1)
     with dask.config.set({"tokenize.ensure-deterministic": False}):
+        # Not idempotent
         assert tokenize(inc) != tokenize(inc)
+        # You can call normalize_token directly
+        assert normalize_token(inc) != normalize_token(inc)
+
     with dask.config.set({"tokenize.ensure-deterministic": True}):
         with pytest.raises(
-            RuntimeError, match=r"Cannot tokenize.*dask\.array\.ufunc.*instead"
+            TokenizationError, match=r"Cannot tokenize.*dask\.array\.ufunc.*instead"
         ):
             tokenize(inc)
 
+    # Test env override
+    assert tokenize(inc, ensure_deterministic=False) != tokenize(
+        inc, ensure_deterministic=False
+    )
```

Comment:
Should this be nested within the block above?

Reply:
No. These 3 tests use the same pattern:

```python
with dask.config.set({"tokenize.ensure-deterministic": False}):
    tokenize(...)
with dask.config.set({"tokenize.ensure-deterministic": True}):
    tokenize(...)
tokenize(..., ensure_determinsitic=False)
tokenize(..., ensure_deterministic=True)
```
This makes them impervious to whatever the config is set to globally.

Reply:
My bad, I attributed the inline comment only to this test case, not the one below.

Reply:
I also wasn't sure if this should test overriding "whatever" the config is set to globally or overriding _any possible_ config value.